internal/pkg/template/templates/environment/cf.yml

```diff
@@ -34,21 +34,31 @@ Parameters:
   CreateInternalHTTPSListener:
     Type: String
     AllowedValues: [true, false]
+  AllowVPCIngressForInternalALB:
```

Comment:
Hmm any reason why we want to add a new param instead of using go template? I am worried about the fact that we add a new parameter that's relatively less likely to be used by users.